corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java

```diff
@@ -212,11 +212,11 @@ private void printPayload(Map.Entry<CorfuDynamicKey, CorfuDynamicRecord> entry)
                     .append(JsonFormat.printer().print(entry.getValue().getPayload()));
             System.out.println(builder.toString());
         } catch (Exception e) {
-            log.error("invalid payload: ", e);
+            //log.error("invalid payload: ", e);
```

Comment:
why was this commented out?

Reply:
probably because of noise, let me change it to trace.